docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -3,18 +3,39 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Using OpenID Connect (OIDC) to Protect Web Applications using Authorization Code Flow
+= OpenID Connect (OIDC) authorization code flow mechanism
 
 include::./attributes.adoc[]
 :toc:
 
-This guide demonstrates how to use Quarkus OpenID Connect (OIDC) Extension to protect your Quarkus HTTP endpoints using OpenID Connect Authorization Code Flow supported by OpenID Connect compliant Authorization Servers such as https://www.keycloak.org[Keycloak].
+The Quarkus OpenID Connect (OIDC) extension can protect your Quarkus HTTP endpoints by using the OIDC Authorization Code Flow mechanism supported by OIDC-compliant authorization servers, such as https://www.keycloak.org[Keycloak].
```

Comment:
Subjective, so feel free to toss it, but I might streamline a little...

```suggestion
The Quarkus OpenID Connect (OIDC) extension can protect application HTTP endpoints using the OIDC Authorization Code Flow mechanism supported by OIDC-compliant authorization servers, such as https://www.keycloak.org[Keycloak].
```

Reply:
Thanks @ebullient. I updated per your suggestion, however, for clarify I kept "by" in front of "using" as there is a slight semantic difference when it is omitted, that is, the sentence might be interpreted as HTTP endpoints "that use" the OIDC Authorization Code Flow mechanism rather than the intended meaning.